Assets/Scenes/TutorialLevel.unity

```diff
@@ -8655,7 +8650,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.032, y: 0.24354449}
   m_AnchorMax: {x: 0.18400002, y: 0.47155875}
-  m_AnchoredPosition: {x: 0.000030517578, y: 0}
+  m_AnchoredPosition: {x: 0, y: 0}
```

Comment:
Not sure what is moved here, but I assume that this leaves every object in the correct location if this is a parent object that is moved.

Reply:
Something in the cutscene, `LeftChar` to be precise.